packages/client.api/src/objectSet/ObjectSet.ts

```diff
@@ -201,9 +201,9 @@ export interface ObjectSet<
    * Fetches one object with the specified primary key, without a result wrapper
    */
   readonly fetchOne: Q extends ObjectTypeDefinition<any> ? <
-      L extends ObjectOrInterfacePropertyKeysFrom2<Q>,
-      R extends boolean,
-      S extends false | "throw" = NullabilityAdherenceDefault,
+      const L extends ObjectOrInterfacePropertyKeysFrom2<Q>,
```

Comment:
This is to fix that edge case you found in return statements right?

Reply:
Yeah. This made tsc be less permissive when trying to align a defined return value from a function with the inferred generics it used.